drivers/sx127x/Kconfig

```
@@ -25,33 +25,22 @@ menuconfig MODULE_SX127X
     help
         Only LoRa long range modem is supported at the moment.
 
-choice SX127X_VARIANT
-    bool "Radio variant"
-    depends on MODULE_SX127X
-    default MODULE_SX1272 if HAVE_SX1272
-    default MODULE_SX1276 if HAVE_SX1276
-
 config MODULE_SX1272
     bool "SX1272"
+    select MODULE_SX127X
 
 config MODULE_SX1276
     bool "SX1276"
-
-endchoice
+    select MODULE_SX127X
 
 config HAVE_SX1272
     bool
-    select HAVE_SX127X
+    select MODULE_SX1272
```

Thinking of this again, this should rather be:
```suggestion
    select MODULE_SX1272 if MODULE_NETDEV_DEFAULT
```